rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -2157,7 +2159,7 @@ rocalAudioFileSource(
     auto context = static_cast<Context*>(p_context);
     try {
 #ifdef ROCAL_AUDIO
-        auto [max_frames, max_channels] = evaluate_audio_data_set(StorageType::FILE_SYSTEM, DecoderType::SNDFILE, source_path, "");
+        auto [max_frames, max_channels] = evaluate_audio_data_set(StorageType::FILE_SYSTEM, DecoderType::SNDFILE, source_path, source_file_list_path);
```

Comment:
@SundarRajan28 In the evaluate_audio_data_set, In the ReaderConfig creation the file_list_path is being set to json path, Is the source evaluation working as expected?

Please check this and ensure file_list_path is set correctly in reader config in that function